settings.gradle.kts

```diff
@@ -6,6 +6,17 @@ pluginManagement {
     }
 }
 
+dependencyResolutionManagement {
+    repositories {
+        mavenCentral()
+    }
+    versionCatalogs {
+        create("ktorLibs") {
+            from("io.ktor:ktor-version-catalog:3.2.2")
```

Comment:
I’m also not sure about using version 3.2.2
cc: @sproshev

Reply:
I reverted back to `3.1.3` but when eventually bumping please skip 3.2.0 & 3.2.1... 😅